### DISCUSSION TITLE
Use issue templates for this repository

### DISCUSSION THREAD

**Discussion by pascallj** [chenxiaolong/avbroot#236] [Ideas]
Recently we have been seeing an increase in issues which are useless without requiring more input from the user. It would save everyone's time if the user supplies some basic information from the get-go. Therefore I think it is useful to configure [issue templates/forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) for this repository, so the users are aware which information they have to supply at a minimum.

This is roughly what I had in mind:
- Avbroot support for new device (if the user is new and has never used avbroot before)
  User should supply:
  - Device name and model
  - Link to full OTA from the device manufacturer's website
  - Version of avbroot and OS
  - Full command
  - Full terminal output
- Avbroot stopped working for existing device (due to a new avbroot release or device update)
  User should supply:
  - (same as above)
  - Last version of avbroot/Android which worked with this device
- Other

It might also be worth to hint the user that it is better to post questions in de 'Discussions' section (but maybe that is only my preference).

**Comment by chenxiaolong** [comment]
This is a good idea.

> It might also be worth to hint the user that it is better to post questions in de 'Discussions' section (but maybe that is only my preference).

I'll defer to the community for this one. For me, issues and discussions are essentially the same because my usual workflow is to check my email notifications and open the link from there. I almost never browse the issue/discussion/PR lists.
